src/site/stages/build/process-cms-exports/schemas/output/node-news_story.js

```diff
@@ -11,11 +11,31 @@ module.exports = {
     entityMetatags: { $ref: 'MetaTags' },
     entityUrl: { $ref: 'EntityUrl' },
     fieldAuthor: {
-      oneOf: [{ $ref: 'output/node-person_profile' }, { type: 'null' }],
+      oneOf: [
+        {
+          type: 'object',
+          properties: {
+            title: { type: 'string' },
+            fieldDescription: { type: ['string', 'null'] },
+          },
+        },
+        { type: 'null' },
+      ],
     },
     fieldFullStory: { $ref: 'ProcessedString' },
     fieldImageCaption: { type: ['string', 'null'] },
     fieldIntroText: { type: 'string' },
+    fieldListing: {
+      type: 'object',
+      properties: {
+        entity: {
+          type: 'object',
+          properties: {
+            entityUrl: { $ref: 'EntityUrl' },
+          },
+        },
+      },
+    },
```

Comment:
I would've used `usePartialSchema`, but then it would've formed a circular dependency with the `node-story_listing` output schema.